docs/index.md

```diff
@@ -698,9 +698,9 @@ Capybara matchers are available if the gem is installed:
 ```ruby
 require "view_component/test_case"
 
-class MyComponentTest < ViewComponent::TestCase
```

Comment:
Here the class name was inconsistent with the component name below. I changed both to `example`.